src/dvr/dvr_config.c

```diff
@@ -937,84 +937,84 @@ const idclass_t dvr_config_class = {
     {
       .type     = PT_BOOL,
       .id       = "day-dir",
-      .name     = N_("Make Subdirectories Per Day"),
+      .name     = N_("Make subdirectories per day"),
       .off      = offsetof(dvr_config_t, dvr_dir_per_day),
       .group    = 4,
     },
     {
       .type     = PT_BOOL,
       .id       = "channel-dir",
-      .name     = N_("Make Subdirectories Per Channel"),
+      .name     = N_("Make subdirectories per channel"),
       .off      = offsetof(dvr_config_t, dvr_channel_dir),
       .group    = 4,
     },
     {
       .type     = PT_BOOL,
       .id       = "title-dir",
-      .name     = N_("Make Subdirectories Per Title"),
+      .name     = N_("Make subdirectories per title"),
       .off      = offsetof(dvr_config_t, dvr_title_dir),
       .group    = 4,
     },
     {
       .type     = PT_BOOL,
       .id       = "channel-in-title",
-      .name     = N_("Include Channel Name In Filename"),
+      .name     = N_("Include channel name In file name"),
       .off      = offsetof(dvr_config_t, dvr_channel_in_title),
       .group    = 5,
     },
     {
       .type     = PT_BOOL,
       .id       = "date-in-title",
-      .name     = N_("Include Date In Filename"),
+      .name     = N_("Include date In file name"),
       .off      = offsetof(dvr_config_t, dvr_date_in_title),
       .group    = 5,
     },
     {
       .type     = PT_BOOL,
       .id       = "time-in-title",
-      .name     = N_("Include Time In Filename"),
+      .name     = N_("Include time In file name"),
       .off      = offsetof(dvr_config_t, dvr_time_in_title),
       .group    = 5,
     },
     {
       .type     = PT_BOOL,
       .id       = "episode-in-title",
-      .name     = N_("Include Episode In Filename"),
+      .name     = N_("Include episode in file name"),
       .off      = offsetof(dvr_config_t, dvr_episode_in_title),
       .group    = 5,
     },
     {
       .type     = PT_BOOL,
       .id       = "subtitle-in-title",
-      .name     = N_("Include Subtitle In Filename"),
+      .name     = N_("Include subtitle in file name"),
       .off      = offsetof(dvr_config_t, dvr_subtitle_in_title),
       .group    = 6,
     },
     {
       .type     = PT_BOOL,
       .id       = "omit-title",
-      .name     = N_("Don't Include Title In Filename"),
+      .name     = N_("Don't include title in file name"),
       .off      = offsetof(dvr_config_t, dvr_omit_title),
       .group    = 6,
     },
     {
       .type     = PT_BOOL,
       .id       = "clean-title",
-      .name     = N_("Remove All Unsafe Characters From Filename"),
+      .name     = N_("Remove all unsafe characters from file name"),
       .off      = offsetof(dvr_config_t, dvr_clean_title),
       .group    = 6,
     },
     {
       .type     = PT_BOOL,
       .id       = "whitespace-in-title",
-      .name     = N_("Replace Whitespace In Title with '-'"),
+      .name     = N_("Replace spaces in title with '-'"),
```

Comment:
Does this include tabs? If so, "whitespace" would be more accurate, or "Replaces tabs and spaces in title..."